problem_236_a/Problem.java

```diff
@@ -1,5 +1,22 @@
+import java.util.Scanner;
+import java.util.HashSet;
+
 public class Problem {
   public static void main(String[] a) {
-    
+    Scanner sc = new Scanner(System.in);
```

Comment:
Java 17+ provides keyword "var". You can use this keyword everywhere you create a variable.
But it means that bad names like "sc" should not be used. The easiest solution here is name like "scanner". 
```
var scanner = new Scanner(System.in);
```